config/jobs/kubernetes/test-infra/fejta-bot-periodics.yaml

```diff
@@ -0,0 +1,211 @@
+      volumeMounts:
+      - name: token
+        mountPath: /etc/token
+    volumes:
+    - name: token
+      secret:
+        secretName: fejta-bot-token
+
+- name: periodic-test-infra-stale
+  interval: 1h
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/commenter:v20170808-abf66782
+      args:
+      - |-
+        --query=org:kubernetes
+        -label:lifecycle/frozen
+        -label:lifecycle/stale
+        -label:lifecycle/rotten
+      - --updated=2160h
+      - --token=/etc/token/bot-github-token
+      - |-
+        --comment=Issues go stale after 90d of inactivity.
+        Mark the issue as fresh with `/remove-lifecycle stale`.
+        Stale issues rot after an additional 30d of inactivity and eventually close.
+
+        If this issue is safe to close now please do so with `/close`.
+
+        Send feedback to sig-testing, kubernetes/test-infra and/or [fejta](https://github.com/fejta).
+        /lifecycle stale
+      - --template
+      - --ceiling=10
+      - --confirm
+      volumeMounts:
+      - name: token
+        mountPath: /etc/token
+    volumes:
+    - name: token
+      secret:
+        secretName: fejta-bot-token
+
+- name: issue-creator
+  interval: 24h
+  labels:
+    preset-service-account: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/issue-creator:v20170907-5240cca5
+      args:
+      - --dry-run=false
+      - --alsologtostderr
+      - --org=kubernetes
+      - --project=kubernetes
+      - --token-file=/etc/token/bot-github-token
+      - --test-owners-csv=/test_owners.csv
+      - --triage-window=1
+      - --triage-count=10
+      - --flakyjob-count=3
+      volumeMounts:
+      - name: token
+        mountPath: /etc/token
+    volumes:
+    - name: token
+      secret:
+        secretName: fejta-bot-token
+
+- name: periodic-features-unfreeze
+  interval: 30m
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/commenter:v20170808-abf66782
+      args:
+      - |-
+        --query=repo:kubernetes/features
+        label:lifecycle/frozen
+      - --updated=5m
+      - --token=/etc/token/bot-github-token
+      - |-
+        --comment=Feature issues opened in `kubernetes/features` should never be marked as frozen.
+        Feature Owners can ensure that features stay fresh by consistently updating their states across release cycles.
+
+        /remove-lifecycle frozen
+      - --ceiling=10
+      - --confirm
+      volumeMounts:
+      - name: token
+        mountPath: /etc/token
+    volumes:
+    - name: token
+      secret:
+        secretName: fejta-bot-token
+
+# periodically file / close bugs for repos based on presence of SECURITY_CONTACTS
+- name: periodic-secping
+  interval: 24h
+  decorate: true
+  spec:
+    containers:
+    # image contains tools versioned with k/k master
+    # secping needs go and make in particular
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20180829-29e661965-master
+      command: 
+      - ./experiment/secping.sh
```

Comment:
I don't see this script in https://github.com/kubernetes/test-infra/tree/master/experiment, should I be?

Reply:
script is below in this PR :upside_down_face: https://github.com/kubernetes/test-infra/pull/9244/files#diff-d49a6ba29e4e1b610757ce532be5f6cb